run/GEOS/GEOSCHEMchem_GridComp.rc

```diff
@@ -226,7 +226,7 @@ Bry_Cly_Family_Transport: 1
 # CO production from CO2 photolysis
 #
 CO_production_from_CO2_photolysis: 0
-CO2photolysisFile: /gpfsm/dnb52/projects/p10/dasilva/fvInput/ExtData/chemistry/GEOSCHEMchem/v0.0.0/CHEM_INPUTS/StratChem/SC.J_20_12_79_72_200_45.jpl15.nc4
+CO2photolysisFile: /discover/nobackup/projects/gmao/share/dasilva/fvInput/ExtData/chemistry/GEOSCHEMchem/v0.0.0/CHEM_INPUTS/StratChem/SC.J_20_12_79_72_200_45.jpl15.nc4
```

Comment:
Why are the changes in this file here? Are they reverting changes made by @lizziel in PR https://github.com/geoschem/geos-chem/pull/2249?

Reply:
I plan to do integration test for this PR and also PR #2271.  I just haven't pushed that yet while the HETP (alpha.15) benchmarks are running).

Reply:
PR #2271 is now merged.